src/CoreWCF.Http/tests/SimpleWSHTTPTest.cs

```diff
@@ -68,6 +68,40 @@ public void WSHttpRequestReplyEchoStringTransportSecurity()
             }
         }
 
+        //This test meant to be run locally.
+        //To make this test run, ensure to change the  defaultInactivityTimeout = TimeSpan.FromSeconds(5) in SecuritySessionServerSettings.
+        // [Fact, Description("Demuxer-failure")]
```

Comment:
That value is supposed to be populated from a binding property. You should be able to modify it in this test.